### ISSUE TITLE
Train models

### ISSUE THREAD

**Issue by nolan-cerebras** [ndey96/deep-generator-network#2]
- [x] Write code required in order to train generator and discriminator sequentially
- [x] Train the generator and discriminator

**Comment by nolan-cerebras** [comment]
These links might be helpful:

original implementation: https://lmb.informatik.uni-freiburg.de/resources/binaries/arxiv2016_alexnet_inversion_with_gans/release_deepsim_v0.5.zip

llink to list of papers from authors' group: https://lmb.informatik.uni-freiburg.de/resources/software.php

[unofficial tensorflow implementation](https://github.com/shijx12/DeepSim) uses batchnorm however the original paper makes no mention of it

**Comment by nolan-cerebras** [comment]
code is implemented in the deepsim train.py
